app/controllers/application_controller.rb

```diff
@@ -175,7 +184,12 @@ def extend_session!
   end
 
   def destroy_sso_cookie!
-    cookies.delete(Settings.sso.cookie_name, domain: cookie_domain, secure: Rails.env.production?)
+    cookies.delete(Settings.sso.cookie_name, domain: Settings.sso.cookie_domain)
+  end
+
+  def should_signout_sso?
+    return false unless Settings.sso.cookie_enabled
```

Comment:
When is Settings.sso.cookie_enabled not true?

Reply:
We may want to disable SSO for any number of reasons. It's essentially saying to put it behind a feature toggle.